libs/data-types/src/lib/decorators/ViewModel.ts

```diff
@@ -0,0 +1,29 @@
+import { COSCRAD_DATA_TYPE_METADATA } from '../constants';
```

Comment:
These are pretty much identical (thanks mr. obvious...)

Reply:
I'm not sure what you mean. The view model schema meta data key is no different than the domain model metadata key. The way we tell them apart is which class they are attached to.